### PR TITLE
Reorganize Kubernetes upgrade files 

### PR DESCRIPTION
**Pull request by amanohar**
**What this PR does / why we need it**:
Reorganize Kubernetes upgrade files

---
This change is [Reviewable](https://reviewable.io/reviews/azure/acs-engine/710)

### CODE REVIEW

**Comment by amanohar** [comment]
@acs-bot test this please

File: .gitignore [hunk 1]
```diff
@@ -7,6 +7,7 @@ _input/
 test/user.env
 user.env
 test/acs-engine-test/acs-engine-test
-
-test/acs-engine-test/acs-engine-test.exe
-pkg/operations/junit.xml
\ No newline at end of file
+
+test/acs-engine-test/acs-engine-test.exe
+pkg/operations/junit.xml
```
Comment (colemickens) [inline .gitignore]: how should I be invoking tests such that these are generated? Is a normal `go test ./...` sufficient?

Comment (amanohar) [inline .gitignore]: Yes, both `go test` under the kubernetesupgrade folder or `go test ./...` both generates the file.

**Comment by seanknox** [comment]
Hey @amanohar @colemickens, please make sure PRs have a meaningful explanation for others who don't have shared context around your change set. It's especially important as we are now have a release process. Let me know if you have any questions (here/slack/etc).

**Comment by amanohar** [comment]
@seanknox, detailed context was in this PR: https://github.com/Azure/acs-engine/pull/707
This PR was a simple move of files to a new folder for better grouping/organization so I don't think it needed a more detailed context.

**Comment by seanknox** [comment]
Right, but there is no obvious connection to #707 mentioned here. Git commit messages need to be meaningful too, so devs looking at the git history can understand the nature of a change without having to look manually at every file changed. Git conventions will start being enforced with this PR: https://github.com/Azure/acs-engine/pull/693, FYI.

**Pull request merged**